src/object/torch.cpp

```diff
@@ -51,16 +52,16 @@ Torch::draw(DrawingContext& context)
 {
   if (m_burning)
   {
-    m_flame->draw(context.color(), get_pos(), LAYER_TILES - 1);
+    m_flame->draw(context.color(), get_pos(), m_layer - 1);
```

Comment:
I'm pretty sure `- 1` should be removed here, since with it we're placing torches 1 layer lower, than user means to.